FeedStoreChallenge/Infrastructure/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Alexander on 17/5/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
This class is not meant to be subclassed so you can mark it as final:

```suggestion
final class ManagedCache: NSManagedObject {
```